### ISSUE TITLE
have generic plotting functionality for loss and other metrics

### ISSUE THREAD

**Issue by avinash2692** [IBM/pytorch-seq2seq#9]
Write a generic plotting function for visualizing training parameters. A few helpful links that this could be based on

* Javascript app from Practical pytorch repo : https://github.com/spro/sconce-python
* Tensorboard without tensorflow :  https://github.com/torrvision/crayon or https://github.com/TeamHG-Memex/tensorboard_logger
